pfdp/css/fundraisers.css

```diff
@@ -0,0 +1,69 @@
+.cards {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  /* flex-grow: 1; */
+  /* grid-row-gap: 30px; */
+  /* grid-column-gap: 30px; */
+}
+
+.card {
+  margin: 15px 15px 15px 15px;
+  width: 363px;
+  height: 450px;
+  /* border: 1px solid black; */
+  position: relative;
+}
+
+.card-top {
+  min-height: 50%;
+  width: 100%;
+}
+
+.pill {
+  position: absolute;
+  height: 75px;
+  width: 75%;
+  background-color: var(--red);
+  border-radius: 0px 300px 300px 0px;
+  top: 40%;
+}
+.pill-text {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 29px;
+  display: flex;
+  align-items: center;
+  color: #FFFFFF;
+  padding-top: 20px;
+  padding-left: 50px;
```

Comment:
better put the padding on the parent than the text

Reply:
with the padding on the parent (the pill), it grows horizontally

Reply:
try to add `box-sizing:border-box`